repl/src/test/scala/com/cloudera/livy/repl/Python3InterpreterSpec.scala

```diff
@@ -0,0 +1,212 @@
+/*
+ * Licensed to Cloudera, Inc. under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  Cloudera, Inc. licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.cloudera.livy.repl
+
+import org.apache.spark.SparkConf
+import org.json4s.{DefaultFormats, JValue}
+import org.json4s.JsonDSL._
+import org.scalatest._
+
+import com.cloudera.livy.sessions._
+
+class Python3InterpreterSpec extends BaseInterpreterSpec {
```

Comment:
This is a good start, but it doesn't share any code with the existing tests. It would be better to have an abstract base class for python interpreter tests, and different real implementations for python 2 and python 3; then adding tests for both is much easier - just change one class, and it's also easy to add version-specific tests (which should be very rare).

The 3 classes could all live in the same source file, since I don't expect too much code in the version-specific classes.